server/tests/src/test/java/org/infinispan/server/test/core/ldap/README.md

```diff
@@ -0,0 +1,20 @@
+```bash
+mvn clean install -pl server/testdriver/core -DskipTests
+mvn clean install -s maven-settings.xml -pl server/tests -DskipTests
+```
+
+```bash
+mvn verify -s maven-settings.xml -pl server/tests -Dit.test="AuthorizationLDAPIT,AuthenticationImplicitIT,AuthenticationLDAPIT,AuthenticationMultiEndpointIT"
+```
+```
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 202, Failures: 0, Errors: 0, Skipped: 0
+```
+
+```bash
+mvn verify -s maven-settings.xml -pl server/tests -Dit.test="AuthorizationLDAPIT,AuthenticationImplicitIT,AuthenticationLDAPIT,AuthenticationMultiEndpointIT" -Dorg.infinispan.test.ldapServer=empty -fae
+```
+```
+[ERROR] Tests run: 202, Failures: 14, Errors: 29, Skipped: 0
```

Comment:
@tristantarrant why when using an LDAP server that does nothing. Do we still have test passing?
For the tests above `AuthorizationLDAPIT,AuthenticationImplicitIT,AuthenticationLDAPIT,AuthenticationMultiEndpointIT`, Do we still need the LDAP server?